src/test/pybind/test_rbd.py

```diff
@@ -2882,6 +2882,38 @@ def test_group_image_remove(self):
         with Image(ioctx, image_name) as image:
             eq(0, image.op_features() & RBD_OPERATION_FEATURE_GROUP)
 
+    def test_group_snap_get_info(self):
+        self.image_names.append(create_image())
+        self.image_names.sort()
+        self.group.add_image(ioctx, self.image_names[0])
+        self.group.add_image(ioctx, self.image_names[1])
+        pool_id = ioctx.get_pool_id()
+
+        self.group.create_snap(snap_name)
+
+        snap_info_dict = self.group.get_snap_info(snap_name)
+
+        image_names = []
+        img_snap_keys = sorted(['pool_id', 'snap_id', 'image_name'])
+        assert sorted(snap_info_dict.keys()) == (
+            sorted(['id', 'name', 'state', 'image_snap_name', 'image_snaps']))
+        assert snap_info_dict['name'] == snap_name
+        for image_snap in snap_info_dict['image_snaps']:
+            assert sorted(image_snap.keys()) == img_snap_keys
+            assert image_snap['pool_id'] == pool_id
+            image_names.append(image_snap['image_name'])
+            with Image(ioctx, image_snap['image_name']) as member_image:
+                snaps = [snap for snap in member_image.list_snaps()]
+            assert len(snaps) == 1
+            assert snaps[0]['name'] == snap_info_dict['image_snap_name']
+            assert snaps[0]['id'] == image_snap['snap_id']
+
+        assert sorted(image_names) == self.image_names
+
+        self.group.remove_snap(snap_name)
```

Comment:
Missing a negative test for a non-existing snapshot, expecting an exception.  One passing `snap_name` could be added here -- otherwise the cleanup could be omitted entirely.